src/bodies.cpp

```diff
@@ -963,6 +963,12 @@ const EigenSTL::vector_Vector3d& bodies::ConvexMesh::getScaledVertices() const
   return scaled_vertices_ ? *scaled_vertices_ : getVertices();
 }
 
+const EigenSTL::vector_Vector4d& bodies::ConvexMesh::getPlanes() const
+{
+  static const EigenSTL::vector_Vector4d empty;
```

Comment:
use an if statement and only allocate ``empty`` if there is no ``mesh_data_``

Reply:
It is static, so does it not only get allocated once? Also this style is used throughout the package.

Reply:
This is fine as is.